src/Sdk/Subscriptions/types.ts

```diff
@@ -0,0 +1,75 @@
+import { Contact, UnsubscribeReason, UploadcareImage } from '../../types';
+
+export interface NewsroomSubscribeRequest {
+    email_address: string;
+    locale?: string;
+    url?: string;
+    visitor_uid?: string;
+    session_uid?: string;
+    comment?: string;
+    contact?: {
+        contact_type: Contact['contact_type'];
+        /**
+         * Required only when contact_type === 'organisation'
+         */
```

Comment:
This and other similar comments can be expressed with TS:

```tsx

interface Contact {
    contact_type: ContactType;
    first_name: string;
    company_name: string;
}

interface NewsroomSubscribeRequest<Type extends Contact['contact_type']> {
    /* ... */
    contact?: (Type extends 'person' ? Person : Organisation) & {
        // common props go here
        languages?: string[];
    };
}

interface Person {
    first_name: Contact['first_name'];
    /* other person-specific attributes go here */
}

interface Organisation {
    company_name: Contact['company_name'];
    /* other organisation-specific attributes go here */
}

// examples:
const x: NewsroomSubscribeRequest<ContactType.PERSON> = {
    contact: {
        languages: [], // ok
        company_name: 'asd', // error
        first_name: 'asd', // ok
    },
};

const y: NewsroomSubscribeRequest<ContactType.ORGANISATION> = {
    contact: {
        languages: [], // ok
        first_name: 'asd', // error
        company_name: 'asd', // ok
    },
};

```

Reply:
@kamilmielnik thank you for this tip/suggestion!

Please check the changes. 

I also added `contact_type` fields to `Person` and `Organisation` interfaces. Also check how I specified type of `payload` parameter of `subscribeToNewsroom` method.